pkg/registry/apis/provisioning/register.go

```diff
@@ -447,5 +446,35 @@ spec:
 		sub.Get.Tags = []string{"API Discovery"} // sorts first in the list
 	}
 
+	for _, sub := range oas.Paths.Paths {
+		paramsOrNil := func(o *spec3.Operation) []*spec3.Parameter {
+			if o == nil {
+				return nil
+			}
+			return o.Parameters
+		}
+		for _, params := range [][]*spec3.Parameter{
+			sub.Parameters,
+			paramsOrNil(sub.Get),
+			paramsOrNil(sub.Delete),
+			paramsOrNil(sub.Post),
+			paramsOrNil(sub.Patch),
+			paramsOrNil(sub.Put),
+			paramsOrNil(sub.Trace),
+			paramsOrNil(sub.Head),
+			paramsOrNil(sub.Options),
+		} {
+			if params == nil {
+				continue
+			}
+
+			for _, p := range params {
+				if p.ParameterProps.Name == "namespace" && p.Schema.Default == nil {
+					p.Schema.Default = "default"
```

Comment:
the value really depends on the user... "default" is true on-prem when org=1

I tried to have the swagger UI fill it in automagicly:
https://github.com/grafana/grafana/blob/main/public/swagger/SwaggerPage.tsx#L65

it seems to work most of the time, but not in our branch -- i'll dig more tomorrow

Reply:
hmm! Would you be against doing this for now anyways, considering we're primarily targetting on-prem? :)

Reply:
yes -- think we can fix this so it shows the right thing.  Not having the right one in cloud blows.  For example:
https://play.grafana.org/swagger?api=banners.grafana.app-v0alpha1#/AnnouncementBanner/listNamespacedAnnouncementBanner

<img width="736" alt="image" src="https://github.com/user-attachments/assets/db637512-f16c-482a-80ea-83c3d16913c4">

sucks when you don't know this is `stacks-35611`!

Reply:
that's fair! I'll deal with the bad default for now, then 😁 

I might take a look at the frontend soon, though I'm not huge on frontend code 😅